.circleci/ds005_outputs.txt

```diff
@@ -47,6 +47,7 @@ fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_MELODICmix.tsv
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-fsaverage5.L.func.gii
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-fsaverage5.R.func.gii
 fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_brainmask.nii.gz
+fmriprep/sub-01/func/sub-01_task-mixedgamblestask_run-01_bold_space-MNI152NLin2009cAsym_preproc.dtseries.nii
```

Comment:
i'm not sure the `+` character is allowed, but the general idea sounds ok for fmriprep. for cifti itself, i could craft up data that could be in any number of spaces.

Reply:
how about `space-MNI152NLin2009cAsym_surf-fsaverage5`?

also, keep in mind both spaces are embedded in the metadata
https://github.com/mgxd/fmriprep/blob/ed8b6424b7e971f0b5eb33daed099b6d18142cb4/fmriprep/interfaces/cifti.py#L222-L226

Reply:
or `space-multiple` and the interpretation would be left to the filetype/file+json.

Reply:
@mgxd surf isn't yet a valid entity key, and I'd rather not proliferate those more than we have to. Space currently can indicate either volumetric or surface spaces, and there's not a principled reason that I can see to choose one over the other as the type of space that gets an additional key.

@satra You're right that `+` is not currently supported... I think labels are defined to be alphanumeric. I would be okay with trying to push to enable `+`, or we can just come up with another name to refer to this space. I'm reluctant to use `space-multiple`, as that isn't any more informative than `.dtseries.nii`, which indicates that it's a CIFTI, and thus the spaces can be retrieved from metadata. We would not be able to support multiple CIFTI output spaces, since the filenames would collide.

If we're okay with this, and want to rely on metadata, I would suggest dropping `space-` entirely. Alternately, let's come up with a name that indicates the spaces are `MNI152NLin2009cAsym` and `fsaverage5`, and add it to the derivatives spec.

Reply:
As I think more about this, I find the `+` approach least objectionable. I'm going to propose to the BIDS discussion list permitting `+` as a delimiter in cases where multiple values reasonably apply to a key.

Reply:
i generally prefer that we don't encode metadata in filenames (unless it's a 90% usecase that would make it more useful than not). there was a discussion on the mailing list on separators. may be worthwhile to look that one up.

in this particular case, since one could generate cifti files in different combination of spaces, it would be useful to encode the difference in the filename. how that happens, i'm a bit more ambivalent about.

Reply:
linking further discussion: https://groups.google.com/forum/#!topic/bids-discussion/ASHycf6JB_0